service/server.py

```diff
@@ -301,7 +281,6 @@ def get_property_address_index_polygon(geometry_data):
 
 
 def _get_address_search_response(search_term, page_number):
-    search_term = search_term.upper()
```

Comment:
I hated the statefulness that this line represented. But doesn't removing it break anything?